lib/launcher.ts

```diff
@@ -296,9 +300,13 @@ let initFn = function(configFile: string, additionalConfig: Config) {
               }
               return cleanUpAndExit(exitCode);
             })
-            .done();
+            .catch(err => setTimeout(() => {
```

Comment:
👍 makes sense. I'd love to have a chat about alterations we can make to the flow.